cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -103,11 +113,13 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
     return arrow::Status::OK();
   }
 
+  /// Returns the file-pointer offset.
   arrow::Result<int64_t> Tell() const {
     RETURN_NOT_OK(CheckClosed());
     return pos_;
   }
 
+  /// Closes the file stream and deletes the chunks.
```

Comment:
and releases the memory used by the chunks

Reply:
\